docs/en/user_guides/1_config.md

```diff
@@ -369,6 +453,28 @@ model = dict(
   2. `--cfg-options model.data_preprocessor.sigma_range=0,0.05`. Note that **NO** white space is allowed in the specified value.
      In addition, if the original type is tuple, it will be automatically converted to list after this way.
 
+We can check the modified config:
+
+```python
+from mmengine.config import Config
+
+cfg = config.fromfile('pspnet_r50-d8_4xb2-40k_cityscapes-512x1024.py')
+print(cfg.model.data_preprocessor)
```

Comment:
these lines don't apply `--cfg-options `, so why did the value change?

I just suggest we should tell the users `--cfg-options ` is used in train.py and `test.py` scrips and the function of `cfg_options` like what does the cfg include before and after applying the 'cfg options'